src/core/event/sidebar.js

```diff
@@ -86,7 +86,7 @@ export function getAndActive(router, el, isParent, autoTitle) {
       const href = a.getAttribute('href');
       const node = isParent ? a.parentNode : a;
 
-      a.title = a.innerText;
+      a.title = a.title || a.innerText;
```

Comment:
```suggestion
      a.title = a.innerText || a.title;
```

should it be like this ?

Reply:
no, In this case, there's already a title, so we don't need to set it up again.

```markdown
<!-- docs/_sidebar.md -->
* [Home](/)
* [Guide](guide.md "The greatest guide in the world")
```

Reply:
cool